### PR TITLE
[KO] Add `Performance Points/Weighting System`

### PR DESCRIPTION
**Pull request by Sitoria**
@sylphi, need your brain cell *grrrrrrrrrrrrrr*
size:S?

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Performance_points/Weighting_system/ko.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+stub: true
+tags:
+  - bonus pp
+  - 보너스 pp
+---
+
+# 가중치 시스템
+
+**가중치 시스템**은 플레이어의 각 [퍼포먼스 포인트](/wiki/Performance_points) 값에 가중치를 부여하여 [총 퍼포먼스 포인트](/wiki/Performance_points/Total_performance_points)를 결정합니다. 이후 시스템은 2번째 값에는 조금 낮은 가중치를, 3번째 값에는 조금 더 낮은 가중치를, 이렇게 조금씩 값마다 가중치를 하향 조정합니다. 플레이어는 퍼포먼스 포인트 시스템 내에서 포인트를 더 얻기 위해 자신의 플레이 능력을 향상시켜야 합니다. 정확한 퍼포먼스 포인트 계산법은 다음 공식을 참고하세요 :
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Performance_points/Weighting_system/ko.md:10]: 비볐습니다

File: wiki/Performance_points/Weighting_system/ko.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+`총 pp = pp[1] * 0.95^0 + pp[2] * 0.95^1 + pp[3] * 0.95^2 + ... + pp[m] * 0.95^(m-1)`
+
+설명:
+
+- `pp` 는 각 점수의 개별 pp 값을 설명합니다.
+- `pp[n]` 는 점수를 내림차순으로 정렬한 후 `n`번째 점수의 pp 값을 나타냅니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Performance_points/Weighting_system/ko.md:17]: kay

File: wiki/Performance_points/Weighting_system/ko.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- `n`의 범위는 1에서 `m`까지이며, 여기서 `m`은 플레이어가 보유한 스코어의 양으로 정의됩니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Performance_points/Weighting_system/ko.md:18]: got it

File: wiki/Performance_points/Weighting_system/ko.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## 보너스 pp
+
+가중치 시스템은 플레이어에게 많은 수의 스코어를 등록해 최대 416.6667 보너스 퍼포먼스 포인트를 얻을 수 있도록 허용되었습니다. 최대의 보너스 pp를 획득하기 위해서는 약 25,397개의 스코어 등록 후 달성됩니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Performance_points/Weighting_system/ko.md:22]: fixed

File: wiki/Performance_points/Weighting_system/ko.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+# 가중치 시스템
+
+**가중치 시스템**은 플레이어의 각 [퍼포먼스 포인트](/wiki/Performance_points) 값에 가중치를 부여하여 [총 퍼포먼스 포인트](/wiki/Performance_points/Total_performance_points)를 결정합니다. 이후 시스템은 2번째 값에는 조금 낮은 가중치를, 3번째 값에는 조금 더 낮은 가중치를, 이렇게 조금씩 값마다 가중치를 하향 조정합니다. 플레이어는 퍼포먼스 포인트 시스템 내에서 포인트를 더 얻기 위해 자신의 플레이 능력을 향상시켜야 합니다. 정확한 퍼포먼스 포인트 계산법은 다음 공식을 참고하세요 :
+
+`총 pp = pp[1] * 0.95^0 + pp[2] * 0.95^1 + pp[3] * 0.95^2 + ... + pp[m] * 0.95^(m-1)`
+
+설명:
+
+- `pp` 는 각 점수의 개별 pp 값을 설명합니다.
```
[the remark this answers is not in the document]
Comment (Sitoria) [reply wiki/Performance_points/Weighting_system/ko.md:16]: ogay

**Comment by Sitoria** [comment]
d

**Review by TPGPL — APPROVED** [review]

**Pull request merged**